WcaOnRails/app/views/competitions/_nav.html.erb

```diff
@@ -187,6 +187,14 @@
         <% end %>
       <% end %>
 
+      <% if @competition.user_should_post_competition_results?(current_user) %>
+        <%= alert :warning, note: true do %>
+          The results are not submitted yet! Click
```

Comment:
nit: Notice how the delegate report warning is personal? It uses the word "your", which I think gives the reader a clear sense that this is something that is their responsibility.

I might consider rewording this a bit. Maybe this: "Please submit results for this competition. You can do so " ?

Whatever you think is best!